slideview/src/main/java/ng/max/slideview/SlideView.java

```diff
@@ -157,6 +151,43 @@ public void setSlideBackgroundColor(ColorStateList color) {
         Util.setDrawableColor(slideBackground, color.getDefaultColor());
     }
 
+    public void setStokeColor(@ColorInt int color) {
+        Util.setDrawableStroke(slideBackground, color);
+    }
+
+    public void setReverseSlide(boolean reverseSlide) {
```

Comment:
After the field is renamed to `reversed`, this setter should be renamed to `setReversed(boolean reversed)`.